test/terra/backends/aer_simulator/test_noise.py

```diff
@@ -223,18 +223,26 @@ def test_kraus_circuit_noise(self, method, device):
         """Test simulation with Kraus quantum errors in circuit."""
         backend = self.backend(method=method, device=device)
         shots = 1000
-        error1 = noise.amplitude_damping_error(0.05)
-        error2 = error1.tensor(error1)
-
+        error0 = noise.amplitude_damping_error(0.05)
+        error1 = noise.amplitude_damping_error(0.15)
+        error01 = error1.tensor(error0)
```

Comment:
I think it might be good to test an error with the qargs the other way round in the same circuit as well?

Reply:
I think its fine one way, as long as the error rates are asymetric. But I will swap the gates to be applied to reverse order for (ie [1,0] instead of [0, 1])

Reply:
The point was more to ensure that we're _certainly_ passing in one error that acts on `[0, 1]` and one on `[1, 0]` to make sure that the remapping works.